storage/simulation/range.go

```diff
@@ -116,10 +128,59 @@ func (r *Range) splitRange(originalRange *Range) {
 	stores := originalRange.getStores()
 	r.Lock()
 	defer r.Unlock()
-	r.desc.Replicas = desc.Replicas
+	r.desc.Replicas = append([]proto.Replica(nil), desc.Replicas...)
 	r.stores = stores
 }
 
+// prepareActions walks through each replica and determines if any action is
+// required using the allocator.
+func (r *Range) prepareActions() {
+	r.Lock()
+	defer r.Unlock()
+	r.nextActions = make(map[proto.StoreID]nextAction)
```

Comment:
this map appears to mirror r.stores exactly. perhaps the two can be combined into one.